frontend/src/Resolver.tsx

```diff
@@ -48,16 +49,20 @@ const Resolver: React.FC = () => {
 
   useEffect(() => {
     slug && resolve?.send([slug], { defaultCaller: true });
-  // eslint-disable-next-line react-hooks/exhaustive-deps
+    // eslint-disable-next-line react-hooks/exhaustive-deps
   }, []);
 
   return (
-    <div className="App h-screen flex flex-col justify-center">
-      {resolve?.isSubmitting && <Loader message={resolvedUrl ? `Redirecting to ${resolvedUrl}` : ""} />}
+    <div className="flex flex-col justify-center h-screen App">
+      {resolve?.isSubmitting && <Loader message={resolvedUrl ? `Redirecting to ${resolvedUrl}` : ''} />}
       {!resolve?.isSubmitting && resolve?.result?.ok && !resolvedUrl && (
         <div className="text-center">
-          <h1 className="text-2xl font-bold text-center">URL not found for <span className="bg-white/5 rounded-md p-2">{slug}</span></h1>
-          <p className="mt-6">Go back to <a href="/">shortener</a>.</p>
+          <h1 className="text-2xl font-bold text-center">
```

Comment:
there are several empty new lines

Reply:
its the tailwind formatting, I had autoformat on save turned on